packages/yoga/src/Button/Button.theme.js

```diff
@@ -59,7 +59,7 @@ const Button = ({
           color: colors.white,
         },
         disabled: {
-          color: colors.text.disabled,
+          color: colors.deep,
```

Comment:
Should `text.disabled` token be updated with `colors.deep`?

Reply:
@Wendlereis this was discussed with Heitor Rodrigues, and he was responsible for making that bridge with the yoga team. He informed me that for this task the change was expected only for the `button` context and that's why the update was not made on the theme, not to affect other components' contexts.